scania-digital-design-system/tegel
# feat(button): submit | reset | button types

**Describe pull-request** 

1. adding option to set native button 'type' via prop buttonType
2. setting shadowDOM: false - to enable work with form elements
3. setting scoped: true - to prevent overwritting

**Solving issue**  
Fixes: [DTS-2031](https://tegel.atlassian.net/browse/DTS-2031)

**How to test**  
1. Go to Storybook preview
4. Test all design controls to make sure button still appears the same
5. Check if buttonType prop propagates to the <button>.

**Suggested test steps**

**Additional context**  
Need check-in session to speak about native "type" vs one we use in code
